src/com/google/javascript/jscomp/DiagnosticGroups.java

```diff
@@ -37,6 +40,9 @@
   static final DiagnosticType UNUSED =
       DiagnosticType.warning("JSC_UNUSED", "{0}");
 
+  public static final Set<String> wildcardExcludedGroups =
+      new HashSet<String>(Arrays.asList(new String[] { "reportUnknownTypes" }));
```

Comment:
Since this is the only project I work on that uses Guava - I just never think of those methods. Fixed.